README.md

```diff
@@ -79,11 +79,22 @@ This project Iceberg also has modules for adding Iceberg support to processing e
 
 ### Compatibility
 
-Iceberg's Spark integration is compatible with Spark 2.4 and Spark 3.0 using the modules in the following table:
-
-| Iceberg version | Spark 2.4.x   | Spark 3.0.x    |
-| --------------- | ------------- | -------------- |
-| master branch   | spark-runtime | spark3-runtime |
-| 0.9.0           | spark-runtime | spark3-runtime |
-| 0.8.0           | spark-runtime |                |
+Iceberg's Spark integration is compatible with Spark using the modules in the following table:
+
+| Iceberg version | Spark 2.4.x   | Spark 3.0.x    | Spark 3.1.x    | Spark 3.2.x    |
+| --------------- | ------------- | -------------- | -------------- | -------------- |
+| 0.13.0          | spark-runtime | spark3-runtime | spark3-runtime | spark3-runtime |
```

Comment:
> 0.13.0
better to keep 'master branch' for ongoing version and keep all the released version below it.
> spark3-runtime
isn't it spark3.0-runtime, 3.1 and 3.2 instead of just spark3 ?

Reply:
It should be `iceberg-spark-runtime` for 2.4, `iceberg-spark3-runtime` for 3.0, and then version-specific modules for the rest, `iceberg-spark-runtime-3.1_2.12` and `iceberg-spark-runtime-3.2_2.12`.